engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -2092,21 +2072,81 @@ public void populateDAG(LogicalPlan dag)
 
   }
 
+  private void populateModules(AppConf appConf, LogicalPlan dag, Map<OperatorConf, Module> moduleMap)
```

Comment:
This function needs to be changed, I am working on another set of changes on top of this pull request to clean this code. Removed this code for now as this was to handle case where user can specify DAG having modules using properties. This functionality will be added by different pull request on top of this.